grails-app/views/order/_summary.gsp

```diff
@@ -70,8 +70,19 @@
         </g:if>
 
         <g:if test="${orderInstance?.id}">
+            <g:isUserInRole roles="[org.pih.warehouse.core.RoleType.ROLE_APPROVER]">
```

Comment:
Let's add this logic as part of a method called canUserRollbackOrder() or something along those lines. We can add a tag for this as well. Or just throw an exception when rollbackOrderStatus is called when it shouldn't be. We'll need some error handling to display the error message when this exception occurs.